Source/controls/plrctrls.cpp

```diff
@@ -389,18 +383,16 @@ void FindTrigger()
 	for (int i = 0; i < nummissiles; i++) {
 		int mi = missileactive[i];
 		if (missile[mi]._mitype == MIS_TOWN || missile[mi]._mitype == MIS_RPORTAL) {
-			int mix = missile[mi].position.tile.x;
-			int miy = missile[mi].position.tile.y;
-			const int newDdistance = GetDistance(mix, miy, 2);
+			const int newDdistance = GetDistance(missile[mi].position.tile, 2);

```

Comment:
```suggestion
			const int newDistance = GetDistance(missile[mi].position.tile, 2);
```

Reply:
Oh, maybe I should have made it more clear that this is a general issues with the var name. Looks like it was actually there before your changes.